sys/vm/vm_fault.c

```diff
@@ -664,6 +687,13 @@ vm_fault_trap(vm_map_t map, vm_offset_t vaddr, vm_prot_t fault_type,
 			*ucode = BUS_OBJERR;
 			break;
 		case KERN_PROTECTION_FAILURE:
+#if __has_feature(capabilities)
+			if ((fault_type & VM_PROT_WRITE_CAP) != 0)
+				segv_ucode = SEGV_STORETAG;
+			else
+#endif
```

Comment:
Do we need to bother with #if here? We don't in various places elsewhere. It's dead code, but it's a slow path, and should compile just fine.